<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeetaFace2DemoClass</class>
 <widget class="QMainWindow" name="SeetaFace2DemoClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1048</width>
    <height>724</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1048</width>
    <height>724</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SeetaFace2Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,4">
      <item>
       <widget class="QLabel" name="label1">
        <property name="styleSheet">
         <string notr="true">font-size:20px;
border:2px dashed black;
border-radius:10px;
background:rgba(0,255,0,100)</string>
        </property>
        <property name="text">
         <string>拖拽图片到这里</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="7,3">
        <item>
         <widget class="QLabel" name="label2">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:20px;
border:2px dashed black;
border-radius:10px;
background:rgba(0,255,0,100)</string>
          </property>
          <property name="text">
           <string>拖拽图片到这里</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCameraCompare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>摄像头实时比对模式</string>
   </property>
   <property name="toolTip">
    <string>源图片和摄像头采集到的视频帧进行比对</string>
   </property>
  </action>
  <action name="actionTwoPicCompare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>两张图片比对模式</string>
   </property>
   <property name="toolTip">
    <string>从本地文件夹中加载两张图片进行比对</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SeetaFace2Demo.qrc"/>
 </resources>
 <connections/>
</ui>
